src/math/Ray.js

```diff
@@ -468,7 +468,7 @@ THREE.Ray.prototype = {
 			}
 
 			diff.subVectors( this.origin, a );
-			var DdQxE2 = sign * this.direction.dot( edge2.crossVectors( diff, edge2 ) );
+			var DdQxE2 = sign * this.direction.dot( edge2.cross( diff ) );
```

Comment:
I rechecked the Vector3 implementations and you are right.
Up to this day I was led to believe, that cross(v) computes and returns the result, whereas crossVectors(a, b) computes, assigns and returns this.
Apparently, that is not the case.

You live and learn, thanks!